app/questionnaire/placeholder_parser.py

```diff
@@ -8,13 +8,16 @@ class PlaceholderParser:
     final values
     """
 
-    def __init__(self, language, answer_store=None, metadata=None):
+    def __init__(self, language, answer_store=None, metadata=None, schema=None):
```

Comment:
We should keep the order of the args in placeholder_parser/placeholder_renderer the same.